recipes/hissw/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "hissw" %}
+{% set version = "1.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hissw-{{ version }}.tar.gz
+  sha256: 16b5acc3ed1cebf0b48506d3ffe4750b7ff40cca4d1a079de4cc6a9259af6616
+
+build:
+  skip: true  # [win]
```

Comment:
What's the problem with Windows exactly? It would be great if we could leave this as `noarch:python` since it's pure Python and there don't seem to be any selectors on packages..

Reply:
The package is a bit wonky in that it provides a wrapper around some shell scripts so while on paper it is just pure Python, it is platform dependent unfortunately.

In principle, it should be able to at least be built for Windows, but it also pulls in some configuration options from an rc file and in doing so assumes the existence of the `HOME` env variable which does not exist by default on Windows. If it is preferable, I can patch this and cut a new release on PyPI to resolve this rather than skipping the windows build. There's no rush here.

Reply:
If you don't mind doing it, it would be really helpful. Without noarch, the package has many more builds based on permutations of platform + python version.